pwiz_tools/Skyline/TestUtil/AbstractUnitTest.cs

```diff
@@ -117,7 +117,7 @@ protected bool RecordAuditLogs
         private static string[] SmallMoleculeDevelopers = {"BSPRATT"}; 
         protected bool RunSmallMoleculeTestVersions
         {
-            get { return TestContext.GetBoolValue("RunSmallMoleculeTestVersions", false) || SmallMoleculeDevelopers.Any(smd => Environment.MachineName.Contains(smd)); }
+            get { return TestContext.GetBoolValue("RunSmallMoleculeTestVersions", true) || SmallMoleculeDevelopers.Any(smd => Environment.MachineName.Contains(smd)); }

```

Comment:
If we're defaulting this to true then we can drop the special treatment for BSPRATT machines.